src/main/java/com/minecolonies/core/entity/ai/workers/guard/AbstractEntityAIGuard.java

```diff
@@ -724,11 +724,6 @@ public boolean isWithinPersecutionDistance(final BlockPos entityPos, final doubl
      */
     private BlockPos getTaskReferencePoint()
     {
-        final ILocation location = buildingGuards.getRallyLocation();
-        if (location != null)
-        {
-            return buildingGuards.getRallyLocation().getInDimensionLocation();
-        }
```

Comment:
Will that not stop them from rallying etc?

Reply:
no that is handled elsewhere, this is just used for combat chaining which we already handle by default so its just not needed/causes odd behaviour